content/updates/2029-02-06-update.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "~2020.2.6 urbit.org update"
+date = 2020-02-06
+description = "A brief update from the Tlon team for ~2020.2.6."
+slug = "/updates/2020-02-06"
+[extra]
+author = "Corrina"
+ship = "roslet-tanner"
+image = "https://media.urbit.org/site/posts/updates/~2020.02.06-update.jpg"
++++
+
+![pajaro dunes](https://media.urbit.org/site/posts/updates/~2020.02.06-update.jpg)
+
+It’s been a busy 2020 already, and we’re excited about everything that’s going live this year. Here’s the latest from the Tlon team:
+
+1. Last month’s updates to Arvo and Bridge included:
+Fixes related to the network stability vote.
+[Updates](https://github.com/urbit/urbit/pull/1996) to both Gall and Ames (our application sandbox and networking module, respectively).
+[Support in Bridge](https://github.com/urbit/bridge/pull/335) for withdrawing stars from the linear release contracts.
+ 
+2. Gavin (`~ridlur-figbud`) wrote about how we created sigils, the visual representation of Urbit IDs. Read the post [here](https://urbit.org/blog/creating-sigils/). 
+
+3. We’re pleased to announce the latest round of our semiannual gifts to Urbit contributors. It’s our way of rewarding the hard work of contributors to the project over the last six months. To see all the contributions we’ve recognized, take a look at the [History](https://grants.urbit.org/history) section on the Grants website.
+    - ~bannum-magtus has been tireless in their efforts as a community Guide in the #ship-starting-support channel on our [Discord server](https://discord.gg/C9ENTt3), and as an Urbit evangelizer.
```

Comment:
None of the ships in the list are in monospace if I read this right.

Reply:
Ah, duh — easy fix